electron/main.js

```diff
@@ -144,6 +144,8 @@ function showMainWindow() {
     },
   });
 
+  let unread_count = 0;
```

Comment:
But otherwise it will not work.

The window would flash again and again, even if you just switch chats or read a message on another device - which is what's happening right now.